ioztat

```diff
@@ -246,7 +269,7 @@ class Dataset:
         Return a copy of Dataset with values normalized to per-second rates
         """
         if self.timespan:
-            d = copy.copy(self)
+            d = Dataset(self.name)
```

Comment:
This creates a new zero Dataset and then divides all its zeros.

Reply:
Ouch …